napari/layers/labels/labels.py

```diff
@@ -1141,8 +1141,13 @@ def _get_value_ray(
                 start_point, end_point, n_points, endpoint=True
             )
             im_slice = self._slice.image.raw
+            bounding_box = self._display_bounding_box(dims_displayed)
+            # bounding box is with upper limit excluded in the uses below
```

Comment:
Took me a second to parse this in context, so I've made the comment more verbose

```suggestion
            # the display bounding box is returned as a closed interval
            # (i.e. the endpoint is included) by the method, but we need
            # open intervals in the code that follows, so we add 1.
```

Reply:
@brisvag why did you mark this as resolved without commenting or committing?